cockpit(fix): alerts documentation improvements (#5699)
scaleway/docs-content: pages/cockpit/how-to/configure-alerts-for-scw-resources.mdx

Switch between the tabs below to create alerts for a Scaleway Instance, an Object Storage bucket, a Kubernetes cluster Pod, or Cockpit logs.

<Tabs>
    <TabsTab label="Scaleway Instance">
      The steps below explain how to create the metric selection and configure an alert condition that triggers when **your Instance consumes more than 10% of a single CPU core over the past 5 minutes.**

      1. In the query field next to the **Loading metrics... >** button, paste the following query. Make sure that the values for the labels you have selected (for example, `resource_id`) correspond to those of the target resource.
         ```bash
         rate(instance_server_cpu_seconds_total{resource_id="xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"}[5m]) > 0.1
         ```
         <Message type="tip">
            The `instance_server_cpu_seconds_total` metric records how many seconds of CPU time your Instance has used in total. It is helpful to detect unexpected CPU usage spikes.
         </Message>

>       2. In the **Set alert evaluation behavior** section, specify how long the condition must be met before triggering the alert.

Reviewer: ```suggestion
      2. In the **Set alert evaluation behavior** section, specify for how long the condition must be met before triggering the alert.
```

      3. Enter a name in the **Namespace** and **Group** fields to categorize and manage your alert rules. Rules that share the same group will use the same configuration, including the evaluation interval which determines how often the rule is evaluated (by default: every 1 minute). You can modify this interval later in the group settings.
         <Message type="note">
            The evaluation interval is different from the pending period set in step 2. The evaluation interval controls how often the rule is checked, while the pending period defines how long the condition must be continuously met before the alert fires.
         </Message>
      4. In the **Configure labels and notifications** section, click **+ Add labels**. A pop-up appears.
      5. Enter a label and value name and click **Save**. You can skip this step if you want your alerts to be sent to the contacts you may already have created in the Scaleway console.
         <Message type="note">

>             In Grafana, notifications are sent by matching alerts to notification policies based on labels. This step is about deciding how alerts will reach you or your team (Slack, email, etc.) based on labels you attach to them. Then, you can set up rules that define who receives notifications in the **Notification policies** page.

Reviewer: ```suggestion
            In Grafana, notifications are sent by matching alerts to notification policies based on labels. This step determines how alerts will reach you or your team (Slack, email, etc.) based on labels you attach to them. Then, you can set up rules that define who receives notifications in the **Notification policies** page.
```

      7. Optionally, check that your configuration works by temporarily lowering the threshold. This will trigger the alert and notify your [contacts](/cockpit/concepts/#contact-points).
    </TabsTab>
    <TabsTab label="Kubernetes Pod">
      The steps below explain how to create the metric selection and configure an alert condition that triggers when **no new Pod activity occurs, which could mean your cluster is stuck or unresponsive.**

      1. In the query field next to the **Loading metrics... >** button, paste the following query. Make sure that the values for the labels you have selected (for example, `resource_name`) correspond to those of the target resource.
         ```bash
         rate(kubernetes_cluster_k8s_shoot_nodes_pods_usage_total{resource_name="k8s-par-quizzical-chatelet"}[15m]) == 0
         ```
         <Message type="tip">
            The `kubernetes_cluster_k8s_shoot_nodes_pods_usage_total` metric represents the total number of Pods currently running across all nodes in your Kubernetes cluster. It is helpful to monitor current Pod consumption per node pool or cluster, and help track resource saturation or unexpected workload spikes.
         </Message>

>       2. In the **Set alert evaluation behavior** field, specify how long the condition must be true before triggering the alert.

Reviewer: ```suggestion
      2. In the **Set alert evaluation behavior** field, specify for how long the condition must be true before triggering the alert.
```

         </Message>
      2. In the **Set alert evaluation behavior** field, specify how long the condition must be true before triggering the alert.
      3. Enter a name in the **Namespace** and **Group** fields to categorize and manage your alert rules. Rules that share the same group will use the same configuration, including the evaluation interval which determines how often the rule is evaluated (by default: every 1 minute). You can modify this interval later in the group settings.
         <Message type="note">
            The evaluation interval is different from the pending period set in step 2. The evaluation interval controls how often the rule is checked, while the pending period defines how long the condition must be continuously met before the alert fires.
         </Message>
      4. In the **Configure labels and notifications** section, click **+ Add labels**. A pop-up appears.
      5. Enter a label and value name and click **Save**. You can skip this step if you want your alerts to be sent to the contacts you may already have created in the Scaleway console.
         <Message type="note">

>             In Grafana, notifications are sent by matching alerts to notification policies based on labels. This step is about deciding how alerts will reach you or your team (Slack, email, etc.) based on labels you attach to them. Then, you can set up rules that define who receives notifications in the **Notification policies** page.

Reviewer: ```suggestion
            In Grafana, notifications are sent by matching alerts to notification policies based on labels. This step determines how alerts will reach you or your team (Slack, email, etc.) based on labels you attach to them. Then, you can set up rules that define who receives notifications in the **Notification policies** page.
```
